### PR TITLE
Fix empty lines for func definition with ellipsis

### PR DESCRIPTION
**Pull request by blablatdinov**
# I have made things!

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [x] I have added my changes to the `CHANGELOG.md`

## Related issues

#2847 

🙏 Please, if you or your company is finding wemake-python-styleguide valuable, help us sustain the project by sponsoring it transparently on https://opencollective.com/wemake-python-styleguide. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

**Comment by sobolevn** [comment]
@blablatdinov btw, if you have time to check why wps check is failing, it would be great :)

File: wemake_python_styleguide/visitors/ast/function_empty_lines.py [hunk 1]
```diff
@@ -73,7 +74,12 @@ def _is_function_end(
         token: tokenize.TokenInfo,
         function_tokens_exists: bool,
         function_start_column: int,
+        function_start_line: int,
     ) -> bool:
+        is_elipsis = token.string == '...'
+        is_elipsis_end = is_elipsis and token.start[0] == function_start_line
```
Comment (sobolevn) [inline wemake_python_styleguide/visitors/ast/function_empty_lines.py:80]: what about 

```python
def some(
   long_arg1: long_annotation1,
   long_arg2: long_annotation2,
) -> None: ...
```

?

Comment (blablatdinov) [reply wemake_python_styleguide/visitors/ast/function_empty_lines.py:80]: @sobolevn I'm add test for this case. Here `_is_function_end` detect end of function by `DEDENT` token

**Comment by blablatdinov** [comment]
> @blablatdinov btw, if you have time to check why wps check is failing, it would be great :)

Gladly) I suggest doing this in a separate PR

**Comment by sobolevn** [comment]
> I suggest doing this in a separate PR

Sure, let's start with it then.

File: wemake_python_styleguide/visitors/ast/function_empty_lines.py (hunk 1 above)
Comment (blablatdinov) [reply wemake_python_styleguide/visitors/ast/function_empty_lines.py:80]: Maybe this is a bug in the tokenizer. Maybe I can send PR to cpython?

Comment (sobolevn) [reply wemake_python_styleguide/visitors/ast/function_empty_lines.py:80]: Let's first try to create a minimal repro example :)

**Review by sobolevn — APPROVED** [review]
Thanks!

**Pull request merged**